fmf/base.py

```diff
@@ -170,6 +174,17 @@ def _initialize(self, path):
         except ValueError:
             raise utils.FormatError("Invalid version format")
 
+        # Check for the config file
+        config_file_path = Path(self.root) / ".fmf" / "config"
```

Comment:
In general there is no benefit from chaining `/` operators even when parsing on windows paths. I am not sure if it's detrimental in terms of performance though.

Reply:
Thanks for the hint, simplified in 1efb3b5.